TEST_APPS/testcases/test_spi_com/README.md

```diff
@@ -0,0 +1,71 @@
+SPI Communication Test
+=============
+
+Testcases
+---------
+
+### test_spi_com
+
+**Description**  
+This is the SPI communication test which verifies various SPI configuration variants. This test is based on Ice Tea framework. Information how to install and use Ice Tea can be found [here](https://github.com/ARMmbed/icetea).  
+This test transfers data between SPI master and SPI slave.
+Two boards are required with SPI peripheral available and slave/master mode support.    
+
+To run the test using same boards as master and slave perform the following steps:  
```

Comment:
A test setup section, is required here.
This should talk about the pre-requirements of the test. 
e.g. required 2 boards can be different, must support slave mode.
A simple wiring diagram needed here.
information about grand wire connection.
Any other tips about hardware preparation

Reply:
Fixed as suggested.